eng/Signing.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <ItemGroup>
+    <ItemsToSign Include="$(SparkPackagesDir)/**/*.nupkg" />
+    <ItemsToSign Include="$(SparkPackagesDir)/**/*.snupkg" />
+    
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.dll" />
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.Worker.exe" />
+    <ItemsToSign Include="$(SparkWorkerPublishDir)/**/Microsoft.Spark.Worker.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- extend arcade's FileExtensionSignInfo with snupkg information -->
+    <FileExtensionSignInfo Include=".snupkg" CertificateName="NuGet" />
```

Comment:
Do we need to specify the certificate name for .jar files? or is that already built-in into arcade's signtool?

Reply:
Already, built in:

https://github.com/dotnet/arcade/blob/8052d0f693173d9bb99f6fa10611a6f4aafd4eb0/src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj#L34

Reply:
Great!